src/platforms/ios/ioscontroller.swift

```diff
@@ -185,6 +185,10 @@ public class IOSControllerImpl : NSObject {
             tunnel.isEnabled = true
 
             return tunnel.saveToPreferences { saveError in
+                // User made a selection, at this point it is safe to run retries via
+                // ConnectionManager::startHandshakeTimer()
+                allowVpnConfigOptionSelectedCallback()
```

Comment:
Consider if something like onTunnelConfigurationCompleted() is a better name. Also, could the comment above be clarified a bit -- it is bit cryptic IMO.